src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerLimits.cs

```diff
@@ -23,6 +23,10 @@ public class KestrelServerLimits
         // Matches the default LimitRequestFields in Apache httpd.
         private int _maxRequestHeaderCount = 100;
 
+        // Matches default HttpRuntimeSection.MaxRequestLength.
+        // https://msdn.microsoft.com/en-us/library/system.web.configuration.httpruntimesection.maxrequestlength%28v=vs.100%29.aspx
+        private long _maxRequestBodySize = 4 * 1024 * 1024;
```

Comment:
IIS7 raised this to 28.6MB
https://www.iis.net/configreference/system.webserver/security/requestfiltering/requestlimits